webshopping/lolshopping/models.py

```diff
@@ -62,6 +65,12 @@ def has_perm(self,perm,obj=None):
     def has_module_perms(self, app_lable):
         return True
 
+    # def save(self, *args, **kwargs):
+    #     try:
+    #         self.new_salt = os.urandom(5)
+    #         super().save(*args, **kwargs)
+    #     except IntegrityError:
+    #         self.save(*args, **kwargs)
```

Comment:
Why is this here?

Reply:
Was there at first but later find a conflicted issue, so I moved it to somewhere else. but will delete it